src/components/Members/Members.js

```diff
@@ -51,15 +49,16 @@ class Members extends Component <Props> {
 
   /**
  * Renders Responsive view of Member's body page.
- * @return {renderized_component} Heder, grid, join us section and footer.
+ * @return {renderized_component} Heder, grid and join us section.
  */
   render() {
     document.title = 'RoBorregos | Members'
     const inactive = this.members.filter((member) => member.status === 'inactive').sort((a, b) => b.id - a.id)
     const active = this.members.filter((member) => member.status === 'active'
       || member.status === 'comitee').sort((a, b) => a.id - b.id)
     return (
-      <div className="members-container" data-testid="members-container">
+      // Warning: data-testid="members-container" prop was removed, add it if needed
```

Comment:
This is fine, the test has snapshots and checks the existence of the other components; you can delete that warning.